src/main/java/models/SpanishSuit.java

```diff
@@ -0,0 +1,10 @@
+package models;
+
+import java.io.Serializable;
+
+/**
+ * Created by Mihai on 2/15/2016.
+ */
+public enum SpanishSuit implements Serializable{
```

Comment:
Since we have the SpanishSuit enum class, I feel like it would make more sense to rename the Suit class to EnglishSuit to make it easier to understand and document.